docs/doxygen/demos/demos_subpages.txt

```diff
@@ -0,0 +1,43 @@
+/**
+@page mqtt_demo_basic_tls Basic TLS Demo
+@brief Uses OpenSSL to establish TLS connection with server authentication, and demonstrates QoS 2 implementation.
+
+This demo demonstrates the subscribe-publish workflow of MQTT. After subscribing to a single topic filter, it publishes to the same topic and waits for receipt of that message to be returned from the server. Once the incoming message is received, the demo publishes an acknowledgement message back to the MQTT server, and the QoS 2 workflow is continued and completed to guarantee the delivery of a message.
+
+Messages in this demo are sent at QoS 2, which guarantees exactly one delivery according to the MQTT spec. See the demo workflow below:
+
+@image html basic_tls.png "Basic TLS Demo Workflow" width=90%
```

Comment:
Did you use drawio to create these? Could you export them using **File > Export as > PNG...** so that there is not a graph in the background?